docs/index.rst

```diff
@@ -10,22 +10,49 @@ eProsima Integration Service Documentation
     :alt: eProsima
     :target: http://www.eprosima.com/
 
-| *eProsima Integration Service* is a library based on *Fast RTPS* for creating parameterized communication bridges between different systems, services, and protocols.
-| It is also able to perform transformations over the messages such as customized routing and mapping between input and output attributes or data modification.
-
-|
-| The main features of *Integration Service* are:
-
-* Connects two different domains.
-* Mapping between different data types.
-* User-defined operations over the received messages (:ref:`transformation library`).
-* Communication with other environments, like *ROS2*.
+*eProsima Integration-Services* is a tool based on `SOSS <https://github.com/eProsima/soss_v2>`__ and its
+**System-Handle** `SOSS-DDS <https://github.com/eProsima/SOSS-DDS>`__ to allow intercommunicating any
```

Comment:
`to allow intercommunication between a DDS-based system and any other protocol-based system, including other DDS systems; as well as offering the possibility to integrate them into a larger, even more complex system.`